src/silo/test/amino_acid_insertion_contains.test.cpp

```diff
@@ -40,10 +40,8 @@ const auto DATABASE_CONFIG = DatabaseConfig{
        .primary_key = "primaryKey"}
 };
 
-const auto REFERENCE_GENOMES = ReferenceGenomes{
-   {{"segment1", "A"}, {"segment2", "T"}},
-   {{"gene1", "*"}, {"gene2", "*"}},
-};
+const auto REFERENCE_GENOMES =
+   ReferenceGenomes{{"segment1", "segment2"}, {"gene1", "gene2"}, {"A", "T"}, {"*", "*"}};
```

Comment:
This looks very unintuitive. It's harder to see what belongs together than before.

Reply:
That is true, but I think we have this problem in many places where we create objects in-line

Reply:
At least here the previous version looks a lot more intuitive :)